h5netcdf/tests/test_h5netcdf.py

```diff
@@ -277,7 +280,9 @@ def read_legacy_netcdf(tmp_netcdf, read_module, write_module):
 
 
 def read_h5netcdf(tmp_netcdf, write_module):
-    remote_file = True if tmp_netcdf.startswith(remote_h5) else False
+    remote_file = False
+    if isinstance(tmp_netcdf, str) and tmp_netcdf.startswith(remote_h5):
```

Comment:
I don't know why this code previously used a ternary conditional statement given that `True if x else False` is equivalent to `x`. Here I would definitely just assign the value of `remote_file` directly.